android/build.gradle

```diff
@@ -1,12 +1,20 @@
 buildscript {
     ext.kotlin_version = '1.7.10'
+    // Versions recommended/needed by flutter_local_notifications and background_fetch
+    // These can be upgraded over time.
+    ext {
+        compileSdkVersion   = 33
+        targetSdkVersion    = 33
+        appCompatVersion    = "1.6.1"
```

Comment:
Just curious, but where did "1.6.1" come from?

Reply:
It essentially comes from [this section](https://github.com/transistorsoft/flutter_background_fetch/blob/master/help/INSTALL-ANDROID.md#androidbuildgradle) of the `background_fetch` setup guide.

``` diff
+       appCompatVersion    = "1.4.2"           // or latest
```

Notice it allows for the latest. I believe I ran into an issue and someone suggesting upgrading, so I looked up the latest  [here](https://developer.android.com/jetpack/androidx/releases/appcompat), and it is 1.6.1.

Sorry I don't remember the specific issue that prompted me to upgrade.

Reply:
That's all good! I was just curious